test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
Please add the test.md file to the .gitignore file.

Reply:
Okay thanks, I'll do that!